packages/admin/cms-admin/src/dam/DataGrid/fileUpload/useDamFileUpload.tsx

```diff
@@ -72,6 +75,10 @@ export interface FileUploadValidationError {
     message: React.ReactNode;
 }
 
+interface RejectedFile {
+    file: FileWithFolderPath;
```

Comment:
This should be of type `File`, right? The `FileRejection` type has no `FileWithFolderPath`. Also, it includes `errors`

Reply:
True, I changed the type and omitted the errors